README.md

```diff
@@ -5,100 +5,110 @@ Module to generate migrations based on source CSV files
 
 Main idea here is to **automatically generate migration config based on source csv**.
 
-We could use next ways of organizing source files:
-* Have next structure of folders for CSV files (like we have for default_content):
-```
--content/
---basic_page/
---article/
--taxonomy_term/
---category
---brands
--user/
-...
-```
-* Having the same, but in source filename, like described in https://github.com/davidferlay/skilld-docker-container/tree/replacedefaultcontentbymigrate/web/modules/custom/generic_content_importer#details, for example `node-basic_page.csv`, `taxonomy_term-category.csv`. etc
-
-*We could have language prefix here also, but I'd say that for 1st iteration, we shouldn't care about translation*
-
-**Main idea here - to have strict rules to identify target Entity type and Bundle for source file.**
-
 ------------
-## Fields support
+## Drush command
+
+`drush migrate_generator:generate_migrations %arg`
+
+Where %arg - absolute path to directory with source .csv files.
```

Comment:
Does it search for files in sub-directories ?

Reply:
1. All source files are supposed to be in same directory. So, I'm not sure that we need to support sub-directories.

2. Added example

4. I was just checking `locale:import` command, and they use absolute path too
so I'm not sure that we'd spend time on that. 
Anyway, we can use next way `drush migrate_generator:generate_migrations $(pwd)/web/modules/custom/migrate_generator/example`

6. Command is used only for generating migrations, not import itself.

Reply:
> about 1. All source files are supposed to be in same directory. So, I'm not sure that we need to support sub-directories.

Np, let's just write it down in readme then

Reply:
sure